test/mix_test_watch/files_test.exs

```diff
@@ -0,0 +1,34 @@
+defmodule MixTestWatch.FilesTest do
+  use ExUnit.Case
+  import MixTestWatch.MockFsHelper
```

Comment:
If you use a macro that changes global state like this the tests need to be run synchronously.